src/service.rs

```diff
@@ -1843,6 +1847,10 @@ impl Service {
         }
     }
 
+    pub fn raw_service_handle(&self) -> Security::SC_HANDLE {
```

Comment:
We have a lot of to-raw exposure in this crate already, and we usually name the methods `to_raw` unless I have missed some places where we are inconsistent(?). Can you please follow the same naming convention?

I also think this type of method should either be close to the top or bottom of methods, not somewhere in the middle. Please look at similar examples and try to be consistent with those (if we have a pattern? But I think we do).

Reply:
I found a number of `to_raw`, but in this case, I changed it to mirror the underlying `raw_handle`. If you'd like `to_raw`, I can of course make that change!

Reply:
Private wrapper type `ScHandle` exposes `raw_handle` but a lot of places use `to_raw`. 🤷‍♂️

Reply:
Would you like to switch it `to_raw`? I'm happy to do so if that's your preference!

Reply:
Personally I'd use `to_raw` to keep the interface homogeneous and familiar to consumers.

Reply:
One could argue we should stop we the bespoke methods and just implement https://doc.rust-lang.org/stable/std/os/windows/io/trait.IntoRawHandle.html for the relevant types instead. But that's not going to happen in this PR. For now I think we should just be consistent with what this library usually uses (`to_raw`)